rdt/transformers/pii/anonymizer.py

```diff
@@ -155,10 +156,13 @@ def reset_randomization(self):
 
     def _function(self):
         """Return a callable ``faker`` function."""
```

Comment:
this is misleading. This function doesn't return a callable. It returns the result of calling the callable right?